protoc_plugin/lib/protobuf_field.dart

```diff
@@ -127,18 +127,18 @@ class ProtobufField {
       String valueType = value.baseType.getDartType(fileGen);
       String keyTypeConstant = key.typeConstant;
       String valTypeConstant = value.typeConstant;
+      String valueCreator = (value.baseType.isMessage || value.baseType.isGroup)
+          ? '$valueType.create'
+          : 'null';
+      String valueOf = value.baseType.isEnum ? '$valueType.valueOf' : 'null';
+      String enumValues = value.baseType.isEnum ? '$valueType.values' : 'null';
+      String mapEntryBuildInfo =
+          "new $_protobufImportPrefix.BuilderInfo('entry')\n"
```

Comment:
Could we create the correct name for `entry` (including the package-name)? I'm not sure how important it is, but it seems in the spirit to keep the name that protoc creates for the map entry-message.